src/kvstore.c

```diff
@@ -44,21 +44,6 @@ struct _kvstore {
     void *metadata[];                      /* conditionally allocated based on "flags" */
 };
 
-/* Structure for kvstore iterator that allows iterating across multiple dicts. */
-struct _kvstoreIterator {
-    kvstore *kvs;
-    long long didx;
-    long long next_didx;
-    dictIterator di;
-};
-
-/* Structure for kvstore dict iterator that allows iterating the corresponding dict. */
-struct _kvstoreDictIterator {
-    kvstore *kvs;
-    long long didx;
-    dictIterator di;
-};
```

Comment:
please explain why move these to .h? do we need to access the variables internal?